sentry.client.config.ts

```diff
@@ -7,10 +7,9 @@ import * as Sentry from '@sentry/nextjs'
 Sentry.init({
   dsn: 'https://56a0c37921d966ca831647a0bb314226@o4507393908211712.ingest.de.sentry.io/4507393908605008',
   enabled: process.env.NODE_ENV === 'production',
-
+  integrations: [Sentry.captureConsoleIntegration()],
```

Comment:
https://docs.sentry.io/platforms/javascript/configuration/integrations/captureconsole/

Reply:
wondering how this will show up in Sentry (e.g. does it appear alongside the errors it's capturing?), but we could limit it with specific log levels in the future https://docs.sentry.io/platforms/javascript/configuration/integrations/captureconsole/#options

Reply:
Yea if we see this gets too much, we can limit it. It appears alongside or also as individual errors depending on the type.